### ISSUE TITLE
Reporting = Blocking users and deleting conversations!!

### ISSUE THREAD

**Issue by capensisma** [RocketChat/Rocket.Chat#1364]
Hallo all,

This feature should be added for sure! since last time someone started spamming me in the demo site!

So a Reporting = Blocking users and deleting conversations feature would be amazing!

Thanks

**Comment by geekgonecrazy** [comment]
Reporting should never directly equal blocking.  There needs to be a place to view reported messages. At that screen an Admin can make a decision to block.

**Comment by capensisma** [comment]
yes I agree they are not equal but there should be a way to block and stop people from sending you unwanted messages from people...

So a block action and if possible report to admins so they can ban or remove this user from certain channels or the whole platform!

**Comment by geekgonecrazy** [comment]
related to #503

**Comment by mahal** [comment]
Would love to have a place to view reported messages.

**Comment by stbc** [comment]
+1
